pkgs/tools/social/twitter/semiphemeral/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, click
+, sqlalchemy
+, flask
+, tweepy
+, colorama
+}:
```

Comment:
```suggestion
{ lib
, python3
}:
```

Reply:
thank you, this is really a lot of experience building

Reply:
The reason we do this is so that it's easier to override Python packages. That's done by using
```nix
semiphemeral.override {
  python3 = python3.override {
    packagesOverrides = ...;
  };
}
```

Reply:
I am not sure if I understand.  This does not build. 

```
error: undefined variable 'click' at /home/aj/proj/nix-nixos/nixpkgs/pkgs/tools/social/twitter/semiphemeral/default.nix:16:29
```

Reply:
See the newest review for how to do it.